content/vault/global/partials/policies/list-allowed-parameters.mdx

```diff
@@ -0,0 +1,24 @@
+#### List parameter evaluation
+
+In Vault v1.20 and earlier, when a request parameter carrying a list value matches
+an `allowed_parameters` or `denied_parameters` policy rule, Vault checks for an exact
+match between each allowed or denied value in the policy and the **entire** list value in
+the request parameter. This behavior can produce unexpected results.
```

Comment:
```suggestion
When a request parameter carrying a list value matches an
`allowed_parameters` or `denied_parameters` policy rule, Vault checks for
an exact match between each allowed or denied value in the policy and
the **entire** list value in the request parameter. The fact that Vault
looks for matches to the entire list can produce unexpected results.
```
Style correction: avoid "this" as a pronoun

Since we only include this partial for docs <= 1.21, we don't need to call out the specific versions